runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/runtimes/download/DownloadManagerDownloadExecutor.java

```diff
@@ -37,6 +38,7 @@
 import org.jboss.tools.rsp.server.spi.model.IServerManagementModel;
 import org.jboss.tools.rsp.server.spi.runtimes.AbstractLicenseOnlyDownloadExecutor;
 import org.jboss.tools.rsp.server.spi.runtimes.AbstractStacksDownloadRuntimesProvider;
+import org.jboss.tools.rsp.server.spi.util.StatusConverter;
 import org.jboss.tools.rsp.server.wildfly.beans.impl.IServerConstants;
 import org.jboss.tools.rsp.server.wildfly.runtimes.download.DownloadManagerStateSingleton.DownloadManagerRequestState;
 import org.jboss.tools.rsp.server.wildfly.servertype.IJBossServerAttributes;
```

Comment:
Couldn't we drop the name "Download" in ```DownloadManagerDownloadExecutor``` at least once, given that it's already in the package name?

Reply:
Shouldn't the constants STEP_CREDENTIALS, STEP_TC, STEP_LICENSE be defined in the DownloadManagerState